routes/toDos/index.js

```diff
@@ -0,0 +1,25 @@
+const {
+    toDos,
+    toDoList,
+    dbCall
```

Comment:
ah ecco cos'era quel method! cmq ti consiglio di accedere direttamente al db con la find dentro i controllers.
in questo modo non devi tirarti dietro questi 2 argomenti extra.

Reply:
dbCall e toDoList